### DISCUSSION TITLE
Include non-linearity in the limitations/discussions section

### DISCUSSION THREAD

**Discussion by atg-abhishek** [Green-Software-Foundation/sci#74] [General]
So you make an interesting point @jawache ,in fact, I'd be shocked if the rate wasn't non-linear since the underlying mechanics are. More than a rate then perhaps we should be thinking of a **model** given some expected traffic pattern. This way, the end consumer can clearly see points of inflection and explore re-architecting the solution if need be when they hit a part of the curve that exacerbates their marginal carbon emissions.

_Originally posted by @atg-abhishek in https://github.com/Green-Software-Foundation/software_carbon_intensity/issues/3#issuecomment-880766164_

**Comment by atg-abhishek** [comment]
@Henry-WattTime could you share the presentation that you had shown in last week's meeting? And any other literature that we could use to think more carefully about the non-linearity aspect of how plants are brought on board in the face of increasing demand for electricity?

**Comment by Henry-WattTime** [comment]
This speaks to the dispatch stack from a price based standpoint, but is a good external resource for how wholesale markets are theoretically dispatched: https://www.iso-ne.com/about/what-we-do/in-depth/how-resources-are-selected-and-prices-are-set

**Comment by atg-abhishek** [comment]
Issue can be closed unless there are further comments.
